modules/log-collection-rbac-permissions.adoc

```diff
@@ -0,0 +1,28 @@
+// Module included in the following assemblies:
+//
+// * logging/log_collection_forwarding/log-forwarding.adoc
+
+:_content-type: PROCEDURE
+[id="log-collection-rbac-permissions_{context}"]
+= Authorizing log collection RBAC permissions
+
+In logging 5.8 and newer versions, the Cluster Logging Operator provides `collect-audit-logs`, `collect-application-logs`, and `collect-infrastructure-logs` cluster roles, which correspond to roles that are able to collect audit logs, application logs, and infrastructure logs respectively.
```

Comment:
```suggestion
In logging 5.8 and later, the Cluster Logging Operator provides `collect-audit-logs`, `collect-application-logs`, and `collect-infrastructure-logs` cluster roles, which correspond to roles that enable the Operator (?) to collect audit logs, application logs, and infrastructure logs respectively.
```

Reply:
Thanks, I actually for some reason thought there were IBMSG guidelines against using "later" and "enabled", so I learned something today 😄